packages/n4s/src/rules/isValueOf.ts

```diff
@@ -0,0 +1,13 @@
+import bindNot from 'bindNot';
+import { isNotNullish } from 'isNullish';
+import { isString, isPlainObject } from 'lodash';
+
+export function isValueOf(value: string, objectToCheck: object): boolean {
```

Comment:
Just noticed the type here. Value should not be a string, but `any`, since anything can be an object value

Reply:
please take into account that if we accept any then the following use case will fail:
`
let welcome= {greeting:{ hebrew:'hi', english:'hello'}};
isValueOf({ hebrew:'hi', english:'hello'}, welcome);
`

Reply:
Yes, that makes sense. `{ hebrew:'hi', english:'hello'}` is not a value in the welcome object. There's an identical value there, but it is not the same one. It is enough for this function to check for referential equality, and assume people refer to it correctly with a variable. 

And regardless, even in that case - what about other primitives? In that case it should have been `boolean|null|undefined|symbol|string|number`.

Reply:
I think we can leave out null and undefined since we are checking that it is not nullish and then take the rest.
anyways whatever we decide we can document

Reply:
we set on checking referential equality for non-primitives